src/sbm.jl

```diff
@@ -194,6 +194,8 @@
     waterlevel_land::Vector{T} | "mm"
     # Water level river [mm]
     waterlevel_river::Vector{T} | "mm"
+    # Total water storage (excluding floodplain volume) [mm]
```

Comment:
Same as previous comment, mention that lakes and reservoirs are also not included